src/views/token-swap/components/quote/quote.view.jsx

```diff
@@ -44,19 +44,6 @@ function Quote({
             </p>
           )}
         </div>
-        <div className={classes.rewardCell}>
-          {info.rewards ? (
```

Comment:
Token swap use different rewards. They are not related to this issue

Reply:
Thx! Reverted!